_data/banking.yml

```diff
@@ -998,7 +995,6 @@ websites:
     img: sofi.png
     tfa:
       - sms
-      - software
```

Comment:
https://github.com/2factorauth/twofactorauth/pull/3951/files#r309195970

Reply:
Is that information still accurate? The doc only mentions SMS.

Reply:
🤷‍♂ yeah you're right. Let's keep it as SMS only for now.